packages/site/pages/components/starrating.js

```diff
@@ -80,6 +80,20 @@ export default withServerProps(_ => (
           `<StarRating value={2.5} onChange={val => alert("The new value is: " + val)} />`
         ]}
       />
+
+      <P>
+        A special empty state will be display if the{' '}
```

Comment:
Special *how* or *why* might be useful prose to write.  At least it seems so to me as one to whom the requirement reasoning is unknown.

Reply:
tell me more. i'm not a great author :/

Reply:
i'm gonna ask dmitri why there is a special case

Reply:
Hehe, yeah, I guess we both need more context if we're going to try to give context to other devs using the component.  You're a great author, btw!  This was just a potential enhancement to the "special style" section.